packages/itwinui-react/setupTests.ts

```diff
@@ -6,6 +6,11 @@ import '@testing-library/jest-dom';
 
 window.HTMLElement.prototype.scrollIntoView = () => {};
 window.HTMLElement.prototype.scrollTo = () => {};
+window.ResizeObserver = jest.fn().mockImplementation(() => ({
```

Comment:
This could be useful to mention in migration guide. Just a short note "Since floating-ui uses `ResizeObserver`, you might need to add a mock in your tests.".

Reply:
Will mention this once this is merged.